x/sequencer/keeper/last_sort_key_test.go

```diff
@@ -0,0 +1,63 @@
+package keeper_test
```

Comment:
It should be `package keeper` or different directory

Reply:
That's the convention in `CometBFT` and `CosmosSDK`. For consistency, I'm leaving it unchanged.